src/v2/Components/NavBar/NavBarLoggedInActions.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useContext } from "react"
+import { NavBarNotificationsQueryRenderer, NavBarUserMenu } from "./Menus"
+import { AnalyticsSchema, SystemContext, useTracking } from "v2/System"
+import {
+  BellIcon,
+  Dropdown,
+  EnvelopeIcon,
+  Flex,
+  SoloIcon,
+} from "@artsy/palette"
+import { graphql } from "relay-runtime"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import {
+  NavBarLoggedInActionsQuery,
+  NavBarLoggedInActionsQueryResponse,
+} from "v2/__generated__/NavBarLoggedInActionsQuery.graphql"
+import { isServer } from "lib/isServer"
+import {
+  getConversationCount,
+  getNotificationCount,
+  updateConversationCache,
+  updateNotificationCache,
+} from "./helpers"
+import styled from "styled-components"
+import { themeGet } from "@styled-system/theme-get"
+import {
+  NavBarItemButton,
+  NavBarItemLink,
+  NavBarItemUnfocusableAnchor,
+} from "./NavBarItem"
+
+/** Displays action icons for logged in users such as inbox, profile, and notifications */
+export const NavBarLoggedInActions: React.FC<Partial<
+  NavBarLoggedInActionsQueryResponse
+>> = ({ me }) => {
+  const { trackEvent } = useTracking()
+
+  const hasUnreadNotifications =
+    (me?.unreadNotificationsCount ?? 0) > 0 || getNotificationCount() > 0
+  const hasUnreadConversations =
+    (me?.unreadConversationCount ?? 0) > 0 || getConversationCount() > 0
+
+  updateNotificationCache(me?.unreadNotificationsCount)
+  updateConversationCache(me?.unreadConversationCount)
+
+  return (
+    <>
+      <Dropdown
+        dropdown={<NavBarNotificationsQueryRenderer />}
+        placement="bottom-end"
+        offset={0}
+      >
+        {({ anchorRef, anchorProps, visible }) => (
+          <NavBarItemButton
+            ref={anchorRef as any}
+            active={visible}
+            onClick={() => {
+              trackEvent({
+                action_type: AnalyticsSchema.ActionType.Click,
+                subject: AnalyticsSchema.Subject.NotificationBell,
+                new_notification_count: getNotificationCount(),
+                destination_path: "/works-for-you",
+              })
+            }}
+            {...anchorProps}
+          >
+            <NavBarItemUnfocusableAnchor href="/works-for-you" />
+
+            <BellIcon
+              title="Notifications"
+              // @ts-ignore
+              fill="currentColor"
```

Comment:
What is this `currentcolor`/

Reply:
A [widely supported CSS value](https://caniuse.com/currentcolor) — so it'll inherit a `:hover` color from the parent, which means you don't have to do swap out the fill. IMO this should be the default fill value for all icons.

Reply:
Very cool and hadn't heard of this before